internal/pkg/cli/app_package.go

```diff
@@ -287,13 +287,10 @@ func (o *packageAppOpts) getAppTemplates(env *archer.Environment) (*appCfnTempla
 			ImageRepoURL: repoURL,
 			ImageTag:     o.Tag,
 		}
-		var appStack *stack.LBFargateStackConfig
-		// If the project supports DNS Delegation, we'll also
-		// make sure the app supports HTTPS
-		if proj.RequiresDNSDelegation() {
-			appStack = stack.NewHTTPSLBFargateStack(createLBAppInput)
-		} else {
-			appStack = stack.NewLBFargateStack(createLBAppInput)
+
+		appStack, err := initLBFargateStack(createLBAppInput, proj.RequiresDNSDelegation())
```

Comment:
Any good for us to make this change?

Reply:
I had to make the change because I need a way of mocking the `stack` object. I don't like creating that global function :( so if you have another idea let me know!

Previously, we weren't mocking the object itself and it would actually read the template from disk :|